lib/api/src/modules/stories.ts

```diff
@@ -327,9 +348,53 @@ export const init: ModuleFn = ({
         },
       });
     },
+    fetchStoryList: async () => {
+      // This needs some fleshing out as part of the stories list server project
+      const result = await global.fetch('/stories.json');
+      const storyIndex = (await result.json()) as StoryIndex;
+
+      // We can only do this if the stories.json is a proper storyIndex
+      if (storyIndex.v !== 3) {
+        return;
```

Comment:
Log a warning here at least

Reply:
This happens if you are running a built story book with stories.json on the v6 store. Do we want to log a warning?

Reply:
I think we should log something, even if it's as simple as `Skipping story index with version v${storyIndex.v}`.

Reply:
OK